167_two_sum.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+
+class Solution:
+    def twoSum(self, numbers: List[int], target: int) -> List[int]:
```

Comment:
This solution is wrong, so correct it to be accepted.

Reply:
Let me ask an question. Which mistake did I make a grammatical mistake or a method's problem.